### PR TITLE
Fix pre code

### PR DESCRIPTION
**Pull request by sujan-s**

### CODE REVIEW

**Comment by sujan-s** [comment]
@rosinghal

File: src/components/CodeBlock/CodeBlock.tsx [hunk 1]
```diff
@@ -14,15 +14,15 @@ export const CodeBlock = ({ ref, source, ...props }: CodeBlockProps) => {
         >
             {({ className, style, tokens, getLineProps, getTokenProps }) => (
                 <pre className={className} style={style}>
-                    <code>
+                    {/* <code> */}
```
Comment (rosinghal) [inline src/components/CodeBlock/CodeBlock.tsx]: Remove commented lines

File: src/components/CodeBlock/CodeBlock.tsx [hunk 2]
```diff
@@ -14,15 +14,15 @@ export const CodeBlock = ({ ref, source, ...props }: CodeBlockProps) => {
         >
```
Comment (sujan-s) [inline src/components/CodeBlock/CodeBlock.tsx]: Should it have `language="json"` by default?

Comment (sujan-s) [reply src/components/CodeBlock/CodeBlock.tsx]: @rosinghal

Comment (rosinghal) [reply src/components/CodeBlock/CodeBlock.tsx]: I think it shouldn't because it's always to specify which language are you expecting the output.

**Pull request merged**